### PR TITLE
Fixes Newsletter and the intro text which is not introtext in VBA it is Centralized Content

### PR DESCRIPTION
**Pull request by eselkin**
## Summary

- Updates graphql query. Uses correct source for intro text and updates newsletter content.
- Processes centralized content for intro text.
- Sitewide Facilities

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14933

## Testing done

- Checked content on tugboat. Checked at multiple viewport sizes.

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Benefits mobile  |   Screenshot 2024-02-20 at 9 17 16 PM    |   Screenshot 2024-02-23 at 1 14 34 PM   |
| Benefits desktop |   Screenshot 2024-02-20 at 9 17 50 PM     |   Screenshot 2024-02-23 at 12 40 35 PM   |
| Intro text mobile |  Screenshot 2024-02-20 at 9 17 08 PM | Screenshot 2024-02-20 at 9 06 26 PM |
| Intro text desktop | Screenshot 2024-02-20 at 9 17 40 PM | Screenshot 2024-02-20 at 9 05 47 PM |

## What areas of the site does it impact?

Only VBA pages

## Acceptance criteria

- [x] Centralized content listed displays on a VBA regional office page
  - [x] Benefit office overview
  - [x] [Subscribe to our newsletter about VA benefits](https://public.govdelivery.com/accounts/USVAVBA/subscribers/new) in the social media area
- [x] Requires design review (added to PR)
- [x] Requires accessibility review (added to PR)

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated (N/A)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

View Cleveland Regional Benefit Office on tugboat: https://web-ihqrv6rby5lxlkn8zqdxf9u7lk1aplcp.demo.cms.va.gov/cleveland-va-regional-benefit-office/locations/cleveland-va-regional-benefit-office/

### CODE REVIEW

File: src/site/filters/liquid.js [hunk 1]
```diff
@@ -823,6 +823,7 @@ module.exports = function registerFilters() {
   };
 
   liquid.filters.replace = (string, oldVal, newVal) => {
+    if (!string) return null;
```
Comment (maxx1128) [inline src/site/filters/liquid.js:826]: 👍

**Review by maxx1128 — APPROVED** [review]

**Comment by laflannery** [comment]
@eselkin I know you only added the newsletter link here but can we add `aria-hidden="true"` to all the `<i>` elements in the "Get Updates" section? Would that be a lot of work?
image

**Review by laflannery — APPROVED** [review]
Thank you @eselkin! LGTM

**Comment by thejordanwood** [comment]
@eselkin In the Get updates section for mobile, is it possible to evenly space all of the links? The Facebook and Twitter links look close together in this example. It should have the same spacing as[ VAMC Get updates sections](https://www.va.gov/new-york-harbor-health-care/locations/manhattan-va-medical-center/).

I'm unable to see the styling in your tugboat link, so can't see if this has already been addressed. My computer must be having some issues.

**Review by micahchiang — APPROVED** [review]

**Comment by eselkin** [comment]
The funny thing is I took the Vet Center's component and copied it. I'll take a look at VAMC now to see if that's a better system.

File: src/site/facilities/vba_social_links.drupal.liquid [hunk 2]
```diff
@@ -1,65 +1,99 @@
...
-                {% endif %}
+    {% if fieldGovdeliveryIdEmerg != empty or fieldNews != empty %}
+      <div class="usa-width-one-half social-links">
+        {% if fieldNews != empty %}
+          <div class="social-links-email vads-u-margin-bottom--2">
+            <a
+              class="vads-u-display--flex vads-u-align-items--baseline vads-u-text-decoration--none"
+              rel="noreferrer"
+              href="{{ fieldNews.uri }}">
+              <i class="fas fa-envelope vads-u-margin-right--1" aria-hidden="true"></i>
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/facilities/vba_social_links.drupal.liquid:13]: requested by design.

File: src/site/facilities/vba_social_links.drupal.liquid [hunk 3]
```diff
@@ -1,65 +1,99 @@
...
+          </div>
+        {% endif %}
+
+        {% if fieldGovdeliveryIdEmerg != empty %}
+          <div class="social-links-email vads-u-margin-bottom--2">
+            <a
+              class="vads-u-display--flex vads-u-align-items--baseline vads-u-text-decoration--none"
+              rel="noreferrer"
+              href="https://public.govdelivery.com/accounts/USVHA/subscriber/new?topic_id={{ fieldGovdeliveryIdEmerg }}">
+              <i class="fas fa-envelope vads-u-margin-right--1" aria-hidden="true"></i>
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/facilities/vba_social_links.drupal.liquid:25]: requested by design.

File: src/site/facilities/vba_social_links.drupal.liquid [hunk 4]
```diff
@@ -1,65 +1,99 @@
...
+              </a>
+            </div>
+          {% endif %}
+          {% if fieldFlickr != empty %}
+            <div class="social-links social-links-flickr vads-u-margin-bottom--2">
+              <a
+                class="vads-u-display--flex vads-u-align-items--baseline vads-u-text-decoration--none"
+                rel="noreferrer"
+                href="{{ fieldFlickr.uri }}">
+                <i class="fab fa-flickr vads-u-text-decoration--none vads-u-margin-right--1" aria-hidden="true"></i>
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/facilities/vba_social_links.drupal.liquid:66]: requested by design.

File: src/site/facilities/vba_social_links.drupal.liquid [hunk 5]
```diff
@@ -1,65 +1,99 @@
...
-                        </div>
-                    {% endif %}
-                </div>
+          {% if fieldYoutube != empty %}
+            <div class="social-links social-links-youtube vads-u-margin-bottom--2">
+              <a
+                class="vads-u-display--flex vads-u-align-items--baseline vads-u-text-decoration--none"
+                rel="noreferrer"
+                href="{{ fieldYoutube.uri }}">
+                <i class="fab fa-youtube vads-u-text-decoration--none vads-u-margin-right--1" aria-hidden="true"></i>
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/facilities/vba_social_links.drupal.liquid:90]: requested by design.

File: src/site/facilities/vba_social_links.drupal.liquid [hunk 6]
```diff
@@ -1,65 +1,99 @@
...
-                            </a>
-                        </div>
-                    {% endif %}
+          {% if fieldInstagram != empty %}
+            <div class="social-links social-links-instagram vads-u-margin-bottom--2">
+              <a
+                class="vads-u-display--flex vads-u-align-items--baseline vads-u-text-decoration--none"
+                rel="noreferrer"
+                href="{{ fieldInstagram.uri }}">
+                <i class="fab fa-instagram vads-u-text-decoration--none vads-u-margin-right--1" aria-hidden="true"></i>
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/facilities/vba_social_links.drupal.liquid:78]: requested by design.

File: src/site/facilities/vba_social_links.drupal.liquid [hunk 7]
```diff
@@ -1,65 +1,99 @@
...
+            </a>
+          </div>
+        {% endif %}
+        {% if fieldFacebook != empty %}
+          <div class="social-links social-links-facebook vads-u-padding-bottom--2">
+            <a
+              class="vads-u-display--flex vads-u-align-items--baseline vads-u-text-decoration--none"
+              rel="noreferrer"
+              href="{{ fieldFacebook.uri }}">
+              <i class="fab fa-facebook-square vads-u-margin-right--1" aria-hidden="true"></i>
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/facilities/vba_social_links.drupal.liquid:36]: requested by design.

**Comment by eselkin** [comment]
@thejordanwood Restructured the spacing on the grid pieces in the Social updates section. It had been the same as the VAMC and Vet Center component, but because of the wording the spacing was off. This was because it wasn't guaranteed to be the same text since it can be changed by editors of the centralized. They should be cautious of changing the centralized text though, probably.

**Comment by thejordanwood** [comment]
@eselkin Thank you! This looks good to me.

**Review by thejordanwood — APPROVED** [review]

**Pull request merged**